python/ambassador_cli/ambassador.py

```diff
@@ -352,7 +336,7 @@ def validate(config_dir_path: str):
     Validate an Ambassador configuration. This is an extension of "config" that
     redirects output to devnull and always exits on error.
 
-    :param config_dir_path: Configuration directory to scan for Ambassador YAML files
+    :param config_dir_path: Configuration directory to scan for Ambassador YAML filesZW
```

Comment:
Why the rename on this line?

Reply:
fat fingering it probably and didn't realize it...I will fix that